Nemo11235/CS160-Final-Project
# Hamburger Menu

**Description:**
Implementing the HamburgerMenu navigation sidebar for the website, along with a blur component to darken the rest of the page. The Router is used to navigate from one page to the next, and all paths are connected with the hamburger menu. 

**QA Steps:**
1. On the home page, open the hamburger menu.
2. With the hamburger menu open, close the menu with the 'exit' button at the top left, and by clicking outside the menu.
3. Hover over all navigation options to test for an orange hover effect glow.
4. Click the 'Multiplayer' option, and verify that it leads to the page with '/race'.
5. Open the hamburger menu on the '/race' multiplayer page, and navigate to the "About Us" page.
6. On the 'About Us' page, navigate to the starting home page.
7. Open the hamburger menu, and toggle the 'Dark Theme' switch to test for functionality.
8. On every page of the website, test for the correct navigation of all current implemented pages
